spec/controllers/catalog_controller_spec.rb

```diff
@@ -321,11 +321,4 @@
       end
     end
   end
-
-  describe 'Routing' do
```

Comment:
is there no routing spec?  these went away (and they're not that important) from here but didn't show up anywhere else in the diff.

Reply:
@ndushay converted to a request spec: https://github.com/sul-dlss/preservation_catalog/pull/1266/files#diff-c4260b347cb15fc683b99f7b34e5df04 (why? I thought it was easier to test redirects in a requet spec. I could be convinced otherwise!)

Reply:
@ndushay hold off on  merging, pls. want to consider adding one bit.

Reply:
again, I've approved this, so it's not a big deal to me -- I looked a the request specs, and they aren't the same sort of thing: "this route goes to this controller" ... but I'm not a stickler for routing specs except when I'm flailing with setting up routes.

Reply:
@ndushay Right you are. Routing specs are testing that given path helpers or URLs route to controllers and actions, which this PR does not change. Thanks!